secureli/utilities/usage_stats.py

```diff
@@ -30,10 +32,19 @@ def post_log(log_data: str):
     API_KEY = os.getenv("API_KEY")
 
     if not API_ENDPOINT or not API_KEY:
-        return
-
-    result = requests.post(
-        url=API_ENDPOINT, headers={"Api-Key": API_KEY}, data=log_data
-    )
-
-    return result.text
+        return PublishLogResult(
```

Comment:
Is the logging optional for users? I'm wondering if the failure result message will show for users who don't add the logging.

Reply:
Are you referring to the output printed to the terminal? This code is for the logging to the log file in `.secureli/logs/`